flixel/FlxCamera.hx

```diff
@@ -1525,6 +1525,27 @@ class FlxCamera extends FlxBasic
 		setScale(scaleX, scaleY);
 	}
 	
+	/**
+	 * Helper function to make sure a point will use integer coordinates after
+	 * applying scale.
+	 */
+	public inline function floorPoint(point:FlxPoint):FlxPoint
```

Comment:
Strikes me as a little weird to have these functions as public API, considering they're only used for `FlxTilemap`.. Do you anticipate they'll be useful elsewhere? If not, we could probably just make them `private` and use `@:access`, or move them to `FlxTilemap` alltogether.